### PR TITLE
Update CI README for PR trigger issue

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

**Comment by beeender** [comment]
image

:sweat:

File: concourse/README.md [hunk 1]
```diff
@@ -63,3 +63,11 @@ To test if the webhook works, use `curl` to send a `POST` request to the hook UR
 ```
 curl --data-raw "foo" <hook_url>
 ```
+
+# FAQ
+
+## PR pipeline is not triggered.
+
+The PR pipeline relies on the webhook to detect the new PR. However, due the the limitation of the webhook implemention of concourse, we rely on the push hook for this. It means if the PR is from a forked repo, the PR pipeline won't be triggered immediately. To manually trigger the pipelien, go to https://extensions.ci.gpdb.pivotal.io/teams/main/pipelines/PR:diskquota/resources/diskquota_pr and click `refresh` button there.
```
Comment (xuebinsu) [inline concourse/README.md:71]:
```suggestion
The PR pipeline relies on the webhook to detect the new PR. However, due the the limitation of the webhook implemention of concourse, we rely on the push hook for this. It means if the PR is from a forked repo, the PR pipeline won't be triggered immediately. To manually trigger the pipeline, go to https://extensions.ci.gpdb.pivotal.io/teams/main/pipelines/PR:diskquota/resources/diskquota_pr and click ⟳ button there.
```

**Review by Sasasu — APPROVED** [review]
but I think https://extensions.ci.gpdb.pivotal.io/teams/main/pipelines/PR:diskquota/resources/diskquota_pr is not public...

File: concourse/README.md (hunk 1 above)
Comment (beeender) [reply concourse/README.md:71]: ⟳ nice emoji

**Comment by beeender** [comment]
> but I think https://extensions.ci.gpdb.pivotal.io/teams/main/pipelines/PR:diskquota/resources/diskquota_pr is not public...

I think it is ok ... The URL has permission control

**Comment by higuoxing** [comment]
My comment doesn't relate to this PR. I think it's good idea to put `pending` status in GitHub UI before running tests or it's hard to tell whether the pipeline is triggered.

e.g.,

When the pipeline is triggered, there will be an orange circle on the right hand side of pull-request title.

image


See: https://github.com/higuoxing/diskquota/blob/c37fb943df17b7106ad308281be773248e6c0e84/concourse/pipeline/pr-pipeline.yml#L101

**Comment by beeender** [comment]
> My comment doesn't relate to this PR. I think it's good idea to put pending status in GitHub UI before running tests or it's hard to tell whether the pipeline is triggered.

yup, found the issue, just need like 10 minutes free time to fix it.

**Comment by beeender** [comment]
> > My comment doesn't relate to this PR. I think it's good idea to put pending status in GitHub UI before running tests or it's hard to tell whether the pipeline is triggered.
> 
> yup, found the issue, just need like 10 minutes free time to fix it.

Done. #157

**Review by Sasasu — APPROVED** [review]

**Review by xuebinsu — APPROVED** [review]

**Pull request merged**